conf/modules.config

```diff
@@ -99,6 +99,7 @@ process {
     withName: FUSIONREPORT {
         ext.when         = { !params.skip_vis }
         ext.args         = "--export csv"
+        ext.args         = { params.fusionreport_filter ? "--tool-cutoff 2" : "--tool-cutoff 1"}
```

Comment:
I would suggest to make the `tool-cutoff` a parameter that you can set so that the user can define the threshold. You can of course have the default set to one or two.

Reply:
Will set is as an issue for next version